tools/local-snapshot-runner-app/localSnapshotRunnerApp

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
+require("./dist/localSnapshotRunnerApp.js");
```

Comment:
Should this be in a bin/ folder? See packages\tools\replay-tool\bin, packages\tools\fetch-tool\bin